tfutils/base.py

```diff
@@ -334,10 +336,6 @@ def run(sess,
         - thres_loss (float, default: 100)
             If loss exceeds this during training, HiLossError is thrown
     """
-    # initialize and/or restore variables for graph
```

Comment:
I think what happened was that your testcase probably had dosave or restore = False, and then my code didn't initialize.  But now in the new setup I wrote today, I think it will (or should) always work. 

Could you put your case as test code (e..g like test2.py or something) so that we run that as well as the one I wrote whenever making changes?